### ISSUE TITLE
isTeamMember returns string instead of boolean

### ISSUE THREAD

**Issue by glebviniarskii** [tspascoal/get-user-teams-membership#24]
```
echo "output 1 ${{steps.teamAffiliation.outputs.isTeamMember == 'false'}}"
echo "output 2 ${{steps.teamAffiliation.outputs.isTeamMember == 'true'}}"
echo "output 3 ${{steps.teamAffiliation.outputs.isTeamMember == false}}"
echo "output 4 ${{!steps.teamAffiliation.outputs.isTeamMember}}"
```
![изображение](https://github.com/tspascoal/get-user-teams-membership/assets/109268426/f9cc90b2-18e2-45b9-8704-570792befdfd)
Really curious about this behavior, first noticed it when `if` returned `false` from `false == false` expression in checking conditions for step

**Comment by tspascoal** [comment]
That is correct, outputs are always [strings](https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs).

The behavior for comparisons in expressions is documented [here](https://docs.github.com/en/actions/learn-github-actions/expressions#operators) (see the `Note`)

If you want to force coercion to a boolean you can do something like this

```
echo "output 2 ${{ fromJSON(steps.teamAffiliation.outputs.isTeamMember) == true}}"
```

Some examples

```
echo ${{ false == 'false' }} # false
echo ${{ 'false' == false }} # false
echo ${{ fromJSON('false') == false }} # true
```

So it is always better to compare strings for equality

Apologies for the vagueness in the README that lead to this confusion I will add a note to make it more clear.

**Comment by tspascoal** [comment]
@glebviniarskii created issue #25 to make this clear.

Thank you for bringing this up
